dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05355AddAppsPortletToLayoutTest.java

```diff
@@ -0,0 +1,55 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.util.Logger;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test adding the APPS portlet to the layout
+ * @author jsanca
+ */
+public class Task537000AddAppsPortletToLayoutTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        // Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    private void dropAppPortlet(final DotConnect dotConnect) {
+        try {
+            final String dropAppPortletSQL = "delete from cms_layouts_portlets where portlet_id = 'apps'";
+            dotConnect.executeStatement(dropAppPortletSQL);
+        } catch (Exception e) {
```

Comment:
Codacy found an issue: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6018094)